ui/apps/platform/src/queries/standard.js

```diff
@@ -93,7 +93,10 @@ export const LIST_STANDARD_NO_NODES = gql`
 `;
 
 export const COMPLIANCE_STANDARDS = (standardId) => gql`
-    query complianceStandards_${standardId}($groupBy: [ComplianceAggregation_Scope!], $where: String) {
+    query complianceStandards_${standardId.replace(
+        /\W/g,
```

Comment:
This is okay for a patch.

The permanent change we will make on master, which removes the need for any additional testing:
* Go back in product code to `query complianceStandards(` without suffix.
* Go back in compliance test code to unique method to wait for multiple synonymous GraphQL requests.